addons/purchase_stock/tests/test_reordering_rule.py

```diff
@@ -742,6 +742,7 @@ def test_change_of_scheduled_date(self):
         delivery.scheduled_date += td(days=7)
         orderpoint.invalidate_cache(fnames=['qty_forecast', 'qty_to_order'], ids=orderpoint.ids)
         orderpoint.product_id.invalidate_cache(fnames=['virtual_available'], ids=orderpoint.product_id.ids)
+        orderpoint._compute_qty_to_order()
```

Comment:
Why it is necessary? Shouldn't be handle by the overrides  of create/write ?

Reply:
Yes it can be removed since we switched to the add_to_compute